### PR TITLE
add primery key in out_sql  (issues #80)

### PR DESCRIPTION
**Pull request by zamanimehdi**
This change was made to fix the servant's problem in clickhouse database.

Signed-off-by: zamani <zamani.mehdi@gmail.com>

### CODE REVIEW

File: lib/fluent/plugin/out_sql.rb [hunk 1]
```diff
@@ -69,6 +70,9 @@ def configure(conf)
       def init(base_model)
         # See SQLInput for more details of following code
         table_name = @table
+        if @primary_key!=nil then
```
Comment (ganmacs) [inline lib/fluent/plugin/out_sql.rb:73]: [nits]
```suggestion
        if @primary_key != nil 
```

**Comment by ganmacs** [comment]
ref: https://github.com/fluent/fluent-plugin-sql/issues/80

**Comment by ganmacs** [comment]
LGTM! Could you follow DCO? https://github.com/fluent/fluent-plugin-sql/pull/84/checks?check_run_id=566788197

**Comment by zamanimehdi** [comment]
> 
> 
> LGTM! Could you follow DCO? https://github.com/fluent/fluent-plugin-sql/pull/84/checks?check_run_id=566788197

I followed this instruction but nothing changed

**Comment by ganmacs** [comment]
https://github.com/fluent/fluent-plugin-sql/pull/84/checks?check_run_id=568061844

>Commit sha: e01c99f, Author: zamani, Committer: zamani; The sign-off is missin

**Pull request closed without merging**